rpm/podman.spec

```diff
@@ -211,6 +211,11 @@ when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 %autosetup -Sgit -n %{name}-%{version_no_tilde}
 sed -i 's;@@PODMAN@@\;$(BINDIR);@@PODMAN@@\;%{_bindir};' Makefile
 
+# cgroups-v1 is supported on rhel9
+%if 0%{?rhel} == 9
+sed -i '/Using cgroups-v1 which is deprecated/d' libpod/runtime.go
```

Comment:
This results in the following dead code:
```go
    if !ok {
    }
```
....which compiles fine on my f39 laptop, go 1.21.6, but does anyone know if this is explicitly allowed in Go?

Reply:
I guess I could delete the entire code block to be safe.

Reply:
it should be fine, a linter will complain of course but that shouldn't be a problem for building.

Reply:
Deleting the whole block would be a `sed` nightmare, much worse than dead code. If go is OK with empty clauses, and we can be assured that this won't break in future, I'm fine with this approach. (My suggested alternative would've been `s/Warn/Info/`)

Reply:
If we want to get fancy we can solve this with build tags and the the spec file then only needs to add a single build tag but I am not sure if it is worth the extra work here, the current sed seems good enough to me.

Reply:
Let's defer the problem until it actually _is_ a problem. Leave the `sed` as-is, and worry about it if/when RHEL9 build fails.

Reply:
replies don't refresh automatically for me so I updated the sed to delete the entire block along with a `DELETE ON RHEL9` comment in runtime.go. Lemme know if that works.

Reply:
> Deleting the whole block would be a `sed` nightmare, much worse than dead code. If go is OK with empty clauses, and we can be assured that this won't break in future, I'm fine with this approach. (My suggested alternative would've been `s/Warn/Info/`)

@edsantiago still a couple of failures. Would those go away with `s/Warn/Info` ?

Reply:
Sure it would fix the test as warn is the default level, but if you move it to info it will not be shown by default and thus make this change mostly pointless